### PR TITLE
Eg 84 - Create Integration test

### PR DESCRIPTION
**Pull request by davidrossdegroot**

### CODE REVIEW

File: aggregator/src/test/java/com/softtech/cloud/egis/integration/AggregatorIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        final String requestId = submitRequestEntity.getBody();
+        assertNotNull("requestId cannot be null", requestId);
+        assertEquals(HttpStatus.CREATED, submitRequestEntity.getStatusCode());
+
+        PersonDetailsResponse personDetailsSystemResponse = getAndWaitForPersonDetailsResponse(requestId,
+                SystemType.DOSCCD);
+        assertNotNull(personDetailsSystemResponse);
+        assertEquals(Long.valueOf(requestId), Long.valueOf(personDetailsSystemResponse.getRequestId()));
+        // need to have the script to create 20 CCD records into the database
+//        assertNotNull("No person details found!", personDetailsSystemResponse.getPersonDetailsForSystemType(SystemType.DOSCCD));
```
Comment (thnguyen2) [inline aggregator/src/test/java/com/softtech/cloud/egis/integration/AggregatorIntegrationTest.java]: Assume this is WIP?

File: aggregator/spotless-java-import-order.importorder [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#Organize Import Order
+#Mon Apr 01 20:44:16 EDT 2019
+5=
+4=gov
+3=com
+2=org
+1=javax
+0=java
```
Comment (carlholmes) [inline aggregator/spotless-java-import-order.importorder]: If you're doing spotless, pull in .gitattributes from the egis-model project to make sure endlines are properly respected.

File: aggregator/src/test/java/com/softtech/cloud/egis/integration/AggregatorIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+    private RetryTemplate retryTemplate;
+
+    @Value("${aggregator.host}")
+    private String host;
+
+    final String token = "eyJjdHkiOiJKV1QiLCJlbmMiOiJBMjU2R0NNIiwiYWxnIjoiZGlyIn0..f9k6jbZIOHJoahrE.1-9HnLxjkhzg_RZz-v7Jf7qzsHZj-iSu0eleTbfukjVJclfRBjSpLUhY1e4Pu-dOvupeBAWySNtDNmcnRj2d-7HWjkAN4LcbaxSuI0ht8x3njGkVR2RNwCgUztzPcCygRe42h-N46ga6KGZfJajmasclSEXT5Fr7cm0aWyVMJo1d5opPnoufC2JzLJTJl-LutA68mlCHlCFiYzbQbWwydQt5_VcCsGpKxFlqwg3-VNjOr6948EDMlb_WZoHmU8T4KTORty9DPnn57iaIW1sQEyZIqw6bOR1BflhDvttcXE_hniAcv_NYiHNqd7ZGGC06QwlzVEdqsA.Ae2PPw3nx_N6gMDwuWvRaA";
+
+    BioInfoRequest expectedRequest = new BioInfoRequest();
+
+//    @Ignore("ADIS will not currently return anything to Aggregator")
```
Comment (carlholmes) [inline aggregator/src/test/java/com/softtech/cloud/egis/integration/AggregatorIntegrationTest.java]: You sure you want to comment this `@Ignore` out?  The other test that calls the "getExpected...." method for ADIS is ignored.

File: aggregator/spotless-java-import-order.importorder (hunk 2 above)
Comment (davidrossdegroot) [reply aggregator/spotless-java-import-order.importorder]: that's already done

```
.git/  adisAdapter/  aggregator/  ccddos/  .gitattributes*  .gitignore*  Jenkinsfile  pipeline-config.yml
➜  egis-e2e-performance-test git:(EG-84) cat .gitattributes
* text eol=lf
*.jar binary
*.bat text=auto%                                                                                                                                                                              ➜  egis-e2e-performance-test git:(EG-84) cat ../egis-model/.gitattributes
* text eol=lf
*.jar text=auto
*.bat text=auto%                                                                                                                                                                              ➜
```

File: aggregator/src/test/java/com/softtech/cloud/egis/integration/AggregatorIntegrationTest.java (hunk 3 above)
Comment (davidrossdegroot) [reply aggregator/src/test/java/com/softtech/cloud/egis/integration/AggregatorIntegrationTest.java]: Nothing works anyway. Stage itself is broken. I will wait until that is fixed before merging this.

**Pull request merged**